superset/assets/visualizations/EventFlow.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {
+  App,
+  withParentSize,
+  cleanEvents,
+  TS,
+  EVENT_NAME,
+  ENTITY_ID,
+} from '@data-ui/event-flow';
```

Comment:
it's just a name-spaced package on npm, under the ["@data-ui" user](https://www.npmjs.com/~data-ui) but mainly so all the `@data-ui` packages are in the same repo.

itwill live at [https://github.com/williaster/data-ui/packages/event-flow](https://github.com/williaster/data-ui/tree/chris--event-flow) but I haven't quite merged the chart into the master @data-ui repo, adding one more feature. [this is the (massive 🙈) PR](https://github.com/williaster/data-ui/compare/chris--event-flow?expand=1)